test/unit/spec/function_output/KBaseSets/SetBrowser-spec.js

```diff
@@ -0,0 +1,9 @@
+define(['widgets/function_output/KBaseSets/SetBrowser'], (SetBrowser) => {
+    'use strict';
+
+    describe('The Loader class', () => {
+        it('should be defined', () => {
+            expect(SetBrowser).toBeDefined();
```

Comment:
is there anything else that you can test here? e.g public API? Instantiation? Type of `SetBrowser`?